### ISSUE TITLE
_015_duplicate_parallel_import flaky

### ISSUE THREAD

**Issue by ry** [denoland/deno#6471]
example https://github.com/denoland/deno/runs/806953504
```
2020-06-25T10:48:59.9541990Z failures:
2020-06-25T10:48:59.9542370Z 
2020-06-25T10:48:59.9543520Z ---- _015_duplicate_parallel_import stdout ----
2020-06-25T10:48:59.9544010Z target_dir /Users/runner/runners/2.263.0/work/deno/deno/target/release
2020-06-25T10:48:59.9544430Z root path /Users/runner/runners/2.263.0/work/deno/deno/test_util/..
2020-06-25T10:48:59.9545310Z deno_exe path /Users/runner/runners/2.263.0/work/deno/deno/target/release/deno
2020-06-25T10:48:59.9546350Z target_dir /Users/runner/runners/2.263.0/work/deno/deno/target/release
2020-06-25T10:48:59.9547320Z deno_exe args run --quiet --reload --allow-read 015_duplicate_parallel_import.js
2020-06-25T10:48:59.9547830Z deno_exe tests path "/Users/runner/runners/2.263.0/work/deno/deno/test_util/../cli/tests"
2020-06-25T10:48:59.9548220Z OUTPUT
2020-06-25T10:48:59.9550030Z error: Uncaught TypeError: EOF while parsing a value at line 1 column 0
2020-06-25T10:48:59.9551390Z error: test failed, to rerun pass '-p deno --test integration_tests'
2020-06-25T10:48:59.9551560Z 
2020-06-25T10:48:59.9552250Z OUTPUT
2020-06-25T10:48:59.9561420Z ##[error]thread '_015_duplicate_parallel_import' panicked at 'bad exit code, expected: 0, actual: 1', test_util/src/lib.rs:265:7
2020-06-25T10:48:59.9565100Z stack backtrace:
2020-06-25T10:48:59.9566200Z    0:        0x10b9056ff - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h83d53b696ac99295
2020-06-25T10:48:59.9567050Z    1:        0x10b92ec4e - core::fmt::write::hf81c429634e1f3ed
2020-06-25T10:48:59.9568420Z    2:        0x10b815819 - std::io::Write::write_fmt::h53fe50e3fff0275d
2020-06-25T10:48:59.9569360Z    3:        0x10b8fc68c - std::io::impls::<impl std::io::Write for alloc::boxed::Box<W>>::write_fmt::h352c9db3a02449c0
2020-06-25T10:48:59.9570190Z    4:        0x10b9094da - std::panicking::default_hook::{{closure}}::ha991e4eca34b4afa
2020-06-25T10:48:59.9571000Z    5:        0x10b9091b8 - std::panicking::default_hook::h722aa3f5c1c31788
2020-06-25T10:48:59.9571810Z    6:        0x10b909aa8 - std::panicking::rust_panic_with_hook::h2cd47f71d6d55501
2020-06-25T10:48:59.9572300Z    7:        0x10b909672 - rust_begin_unwind
2020-06-25T10:48:59.9573090Z    8:        0x10b939b2b - std::panicking::begin_panic_fmt::h769fb8929973777e
2020-06-25T10:48:59.9573920Z    9:        0x10b854f91 - test_util::CheckOutputIntegrationTest::run::h4e23869c72d8a4fe
2020-06-25T10:48:59.9574760Z   10:        0x10b785bfb - core::ops::function::FnOnce::call_once::h7311f1ce8926695a
2020-06-25T10:48:59.9575580Z   11:        0x10b83aaf3 - test::run_test::run_test_inner::{{closure}}::hf35455f67ec1e4ed
2020-06-25T10:48:59.9576560Z   12:        0x10b814d1b - std::sys_common::backtrace::__rust_begin_short_backtrace::hffd4a983e423c33e
2020-06-25T10:48:59.9577420Z   13:        0x10b81a1f5 - core::ops::function::FnOnce::call_once{{vtable.shim}}::hd45267100ae6c7ce
2020-06-25T10:48:59.9578270Z   14:        0x10b91174d - std::sys::unix::thread::Thread::new::thread_start::h2b28b74d30bce841
2020-06-25T10:48:59.9578780Z   15:     0x7fff6b557109 - _ZL12preoptimized
2020-06-25T10:48:59.9578880Z 
2020-06-25T10:48:59.9578960Z 
2020-06-25T10:48:59.9579090Z failures:
2020-06-25T10:48:59.9579230Z     _015_duplicate_parallel_import
2020-06-25T10:48:59.9579430Z 
2020-06-25T10:48:59.9579600Z test result: FAILED. 216 passed; 1 failed; 12 ignored; 0 measured; 0 filtered out
```

**Comment by bartlomieju** [comment]
https://github.com/denoland/deno/runs/807079745

**Comment by bartlomieju** [comment]
Fixed in #6475
